mods/cnc/maps/shellmap/map.yaml

```diff
@@ -182,7 +180,7 @@ Actors:
 	Actor40: t06
 		Location: 8,6
 		Owner: Neutral
-	Actor43: afld
+	NodAfld: afld
```

Comment:
We've recently agreed on putting all the actors with custom names at the end of the list. It would be nice if you could do that here, too.